src/lib/core/tests/TestCHIPTLV.cpp

```diff
@@ -230,14 +230,13 @@ void TestDupString(nlTestSuite * inSuite, TLVReader & reader, uint64_t tag, cons
     uint32_t expectedLen = strlen(expectedVal);
     NL_TEST_ASSERT(inSuite, reader.GetLength() == expectedLen);
 
-    char * val = static_cast<char *>(chip::Platform::MemoryAlloc(expectedLen + 1));
+    chip::Platform::ScopedMemoryBuffer<char> valBuffer;
+    char * val = static_cast<char *>(valBuffer.Alloc(expectedLen + 1).Get());
```

Comment:
is the cast needed? the scoped buffer template should already be typed.

same for below

Reply:
Right, was left over from Platform::MemoryAlloc.